specification/servicenetworking/resource-manager/Microsoft.ServiceNetworking/preview/2024-05-01-preview/TrafficController.json

```diff
@@ -0,0 +1,1912 @@
+        "name": "ProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "name": "Provisioning",
+            "value": "Provisioning"
+          },
+          {
+            "name": "Updating",
+            "value": "Updating"
+          },
+          {
+            "name": "Deleting",
+            "value": "Deleting"
+          },
+          {
+            "name": "Accepted",
+            "value": "Accepted"
+          },
+          {
+            "name": "Succeeded",
+            "value": "Succeeded"
+          },
+          {
+            "name": "Failed",
+            "value": "Failed"
+          },
+          {
+            "name": "Canceled",
+            "value": "Canceled"
+          }
+        ]
+      }
+    },
+    "ResourceId": {
+      "type": "object",
+      "description": "Resource ID definition used by parent to reference child resources.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID of child resource."
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "SecurityPolicy": {
+      "type": "object",
+      "description": "SecurityPolicy Subresource of Traffic Controller.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/SecurityPolicyProperties",
+          "description": "The resource-specific properties for this resource.",
+          "x-ms-client-flatten": true,
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ]
+    },
+    "SecurityPolicyConfigurations": {
+      "type": "object",
+      "description": "SecurityPolicyConfigurations Subresource of Traffic Controller.",
+      "properties": {
+        "wafSecurityPolicy": {
+          "$ref": "#/definitions/WafSecurityPolicy"
+        }
+      }
+    },
+    "SecurityPolicyListResult": {
+      "type": "object",
+      "description": "The response of a SecurityPolicy list operation.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The SecurityPolicy items on this page",
+          "items": {
+            "$ref": "#/definitions/SecurityPolicy"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "SecurityPolicyProperties": {
+      "type": "object",
+      "description": "SecurityPolicy Properties.",
+      "properties": {
+        "policyType": {
+          "$ref": "#/definitions/PolicyType",
+          "readOnly": true
+        },
+        "wafPolicy": {
+          "$ref": "#/definitions/WAFPolicy"
+        },
+        "ddosProtectionPlan": {
```

Comment:
Sorry if the API design doc was misleading. We do not plan to include any other policy types than WAF in this API version. The mention of DDoS protection plan was purely speculative, and included in the design doc strictly as an example on how the API may be extended. Let's remove it from the definitions in this PR.

Reply:
sure. My bad, I should have clarified. Removed.